metricbeat/module/etcd/metrics/metrics_test.go

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-// +build !integration
+// +build !integration, !386
```

Comment:
Are these tests failing in all 32-bit platforms or only in Windows?

Reply:
only windows 32 bit

Reply:
Could we explicitly skip the tests only for Windows 32 bits instead of for all 386 platforms? With a comment to know what they are skipped.

Reply:
sure, I have made the changes